access/grpc/grpc.go

```diff
@@ -1129,3 +1129,127 @@ func (c *BaseClient) subscribeEvents(
 
 	return sub, errChan, nil
 }
+
+func (c *BaseClient) SubscribeBlockHeadersFromStartBlockID(
+	ctx context.Context,
+	startBlockID flow.Identifier,
+	blockStatus flow.BlockStatus,
+	opts ...grpc.CallOption,
+) (<-chan flow.BlockHeader, <-chan error, error) {
+	request := &access.SubscribeBlockHeadersFromStartBlockIDRequest{
+		StartBlockId: startBlockID.Bytes(),
+		BlockStatus:  convert.BlockStatusToEntity(blockStatus),
```

Comment:
the server won't accept `entities.BlockStatus_BLOCK_UNKNOWN`. can you check first and return an error if the status isn't sealed or finalized. that will make the error easier to understand for users